### DISCUSSION TITLE
Дока по Nunitabc?

### DISCUSSION THREAD

**Discussion by Git-I985** [pascalabcnet/pascalabcnet#2459] [Q&A]
Хотел уточнить где можно будет про модульные тесты почитать, а то с 3.7 вроде они появились, nunit впихнули, а где почитать не ясно)

**Comment by miks1965** [comment]
[https://drive.google.com/file/d/1D_oPI_lNVNoheYCJjVa1SA2hqJ1iwyjb/view?usp=sharing](url)
Посмотрите тут пока доки нет
